hw/xbox/xid-sdl.c

```diff
@@ -488,7 +488,7 @@ static void usb_xbox_gamepad_realize(USBDevice *dev, Error **errp)
     s->out_state_capabilities.report_id = 0;
 
     /* FIXME: Make sure SDL was init before */
-    if (SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER)) {
+    if (SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER | SDL_INIT_HAPTIC)) {
```

Comment:
This took me an embarrassingly long time to catch. While I'm here maybe I should touch up the `FIXME` above it.

Reply:
The FIXME is still valid for non SDL UIs - we'll have to review if QEMU does the SDL init / if the SDL1 / SDL2 behaviour changed.

I believe SDL2 `SDL_InitSubSystem` is more powerful in that it doesn't need another init before.